docs/developer/metrics/metrics.md

```diff
@@ -134,11 +134,11 @@ Starting in cluster version 2.17, some of the metrics names have been changed. I
 | runai\_node\_gpu\_used\_memory\_bytes | renamed and changed units: "runai\_gpu\_memory\_used\_mebibytes\_per\_node"                           |                                                                                                                        |
 | runai\_node\_total\_memory\_bytes | renamed and changed units: "runai\_gpu\_memory\_total\_mebibytes\_per\_node"                          |                                                                                                                        |
 | runai\_project\_info | labels changed                                                                                        |                                                                                                                        |
-| runai\_active\_job\_cpu\_limits | available also via + renamed to: "runai\_cpu\_limits\_per\_active\_workload"                              | https://app.run.ai/api/v1/workloads/{workloadId}/metrics ; with "CPU\_LIMIT\_CORES" metricType                         |
-| runai\_job\_cpu\_usage | available also via + labels changed                                                                       | https://app.run.ai/api/v1/workloads/{workloadId}/metrics ; with "CPU\_USAGE\_CORES" metricType                         |
-| runai\_active\_job\_memory\_limits | available also via + renamed to: "runai\_memory\_limits\_per\_active\_workload"                           | https://app.run.ai/api/v1/workloads/{workloadId}/metrics ; with "CPU\_MEMORY\_LIMIT\_BYTES" metricType                        |
+| runai\_active\_job\_cpu\_limits | available also via API + renamed to: "runai\_cpu\_limits\_per\_active\_workload"                              | https://app.run.ai/api/v1/workloads/{workloadId}/metrics ; with "CPU\_LIMIT\_CORES" metricType                         |
```

Comment:
@lavianalon You used a "+" symbol. Do you mean "&"? Either way use the full word "and".